src/nni_evaluation_engine.hpp

```diff
@@ -0,0 +1,192 @@
+// Copyright 2019-2022 bito project contributors.
+// bito is free software under the GPLv3; see LICENSE file for details.
+//
+// The NNI Evaluation Engine is an interface between the NNI Engine, that facilitates
```

Comment:
between the NNI engine and what?

Reply:
changed to "an interface **for** the NNI Engine"